### ISSUE TITLE
Animated flamegraphs

### ISSUE THREAD

**Issue by bobrik** [Netflix/flamescope#29]
I read the blog post and found it very relatable:

* https://medium.com/netflix-techblog/netflix-flamescope-a57ca19d47bb

It would be awesome to have animated mode where one could select a range and then replay either step by step or at some reasonable rate (say 20ms per frame, 10 fps).

Flamegraph "bricks" would then shrink and expand with each "frame". This is effectively watching individual flamegraphs one after another, but with smooth transformations.

Hopefully this makes sense.

**Comment by brendangregg** [comment]
Makes sense. We have the data to do that. We could have "step forward" and "step backwards" buttons. Imagine selecting one second, then being able to step forward by one second at a time.

**Comment by spiermar** [comment]
The D3 plugin already supports that and I probably have a script to parse the profiles that way. See http://martinspier.io/d3-flame-graph/live.html
